### ISSUE TITLE
slideshow displays every other image on mobile

### ISSUE THREAD

**Issue by TimeOutOfMind** [blueimp/jQuery-File-Upload#3623]
large image/slideshow works fine on my Win 10 Laptop with Chrome 80.0.

On my LG G6 phone running  Android 8.0.0 and Chrome 80.0.3987.162
it displays only every other image file  (1, 3, 5 ...)  in the slide show and in the tiny circular thumbnails below it.

Thanks!

v10.10.0       commit 9b294373a70058f78cd3ad9955adcb5bdcc01979

**Comment by blueimp** [comment]
Hi @TimeOutOfMind,

the slideshow Gallery is not part of this project, but is hosted here:
https://github.com/blueimp/Gallery
jQuery File Upload just includes the minified script for the Gallery on its demo page.

Were you able to reproduce the issue on the demo page?

**Comment by TimeOutOfMind** [comment]
Thanks for the reply.

I'm wasn't clear which demo page you wanted me to reproduce it on ... so I checked both.

Here? https://blueimp.github.io/jQuery-File-Upload/
Yes - error occurred- I uploaded 5 test images and the carousel would only show 3 images on my phone   #1, #3, #5 

Here? https://blueimp.github.io/Gallery/
No error - it's kind of an apples/oranges comparison, but I counted the same number of images (about 100) in the "carousel image gallery" on both my phone and laptop.

Let me know if there's anything else I can do to troubleshoot.

Thanks!
